src/chrome/content/rules/freedos.org.xml

```diff
@@ -3,7 +3,6 @@
 	<target host="www.freedos.org" />
 	<target host="personal.freedos.org" />
 	<target host="test.freedos.org" />
-	<target host="wiki.freedos.org" />
```

Comment:
Should be moved to a comment: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts